--- qsystem.py	2019-05-28 15:59:47.516917559 -0300
+++ qsystem_.py	2019-05-28 15:59:42.073576836 -0300
@@ -685,8 +685,8 @@
     thisown = property(lambda x: x.this.own(), lambda x, v: x.this.own(v), doc='The membership flag')
     __repr__ = _swig_repr
 
-    def __init__(self, *args):
-        _qsystem.QSystem_swiginit(self, _qsystem.new_QSystem(*args))
+    def __init__(self, nqbits, seed=42, state="vector", init=0):
+        _qsystem.QSystem_swiginit(self, _qsystem.new_QSystem(nqbits, seed, state, init))
     __swig_destroy__ = _qsystem.delete_QSystem
 
     def evol(self, gate, qbit, count=1, inver=False):
